src/doc/layer.cpp

```diff
@@ -540,6 +540,20 @@ void LayerGroup::allTilemaps(LayerList& list) const
   }
 }
 
+std::string LayerGroup::visibleLayerHierarchyAsString(std::string indent) const
```

Comment:
This might be not correct place for this function, I have doubts but I couldn't find a good place though